Source/Blazorise/Components/Steps/Steps.razor.cs

```diff
@@ -163,37 +192,14 @@ internal int IndexOfStep( string name )
     protected string ContentClassNames => ContentClassBuilder.Class;
 
     /// <summary>
-    /// Gets or sets currently selected step name.
+    /// Gets currently selected step name.
     /// </summary>
-    [Parameter]
-    public string SelectedStep
-    {
-        get => state.SelectedStep;
-        set
-        {
-            // prevent steps from calling the same code multiple times
-            if ( value == state.SelectedStep )
-                return;
-
-            var allowNavigation = NavigationAllowed?.Invoke( new StepNavigationContext
-            {
-                CurrentStepName = state.SelectedStep,
-                CurrentStepIndex = IndexOfStep( state.SelectedStep ),
-                NextStepName = value,
-                NextStepIndex = IndexOfStep( value ),
-            } ) ?? true;
-
-            if ( allowNavigation == false )
-                return;
+    public string SelectedStep => state.SelectedStep;
 
-            state = state with { SelectedStep = value };
-
-            // raise the changed notification
-            SelectedStepChanged.InvokeAsync( state.SelectedStep );
-
-            DirtyClasses();
-        }
-    }
+    /// <summary>
+    /// Initial step to go on the component initialization
+    /// </summary>
+    [Parameter] public string InitialStep { get; set; } = string.Empty;
```

Comment:
I don't think having an `InitialStep` makes much sense. We can always internally store the SelectedPage during the component initialization lifecycle and use it for later where needed.

Reply:
Since the `SelectedStep` is getter-only, this is the only way how to setup the initial value on component itself. 
But this is almost the same discussion as the one above - about having the `SelectedStep` two-way bindable or not.

Reply:
No longer an issue with SelectedStep being two-way bindiable.